src/manifest.rs

```diff
@@ -108,6 +110,11 @@ fn merge_dependencies(old_dep: &mut toml::value::Value, new: &Dependency) {
             toml::Value::Table(ref mut n) => old.append(n),
```

Comment:
We're not calling `print_upgrade_if_necessary` in this branch. `cargo upgrade` currently doesn't hit it, but `cargo add` does (which risks making `cargo add`'s behaviour inconsistent).

Might it make sense to call `print_upgrade_if_necessary` from [`update_table_entry`](https://github.com/Eijebong/cargo-edit/blob/cargo_update_verbose/src/manifest.rs#L321) instead? It would make the code responsible for informing the user separate from the code which merges the new dependency into the old one correctly. It might also be easier to extend to fully cover `cargo add` in future (i.e. by adding similar appropriate print functions to [`insert_table_entry`](https://github.com/Eijebong/cargo-edit/blob/cargo_update_verbose/src/manifest.rs#L288)).